analysis_ready_planetscope/truecolor_cloudMasked.js

```diff
@@ -0,0 +1,24 @@
+//VERSION=3
+
+function setup() {
+    return {
+        input: [{
+            bands: ["blue", "green", "red", "cloud_mask"],
+        }],
+        output: {
+            bands: 4
+        }
+    };
+}
+
+let factor = 1/2000; 
+function evaluatePixel(sample) {
+    if (sample.cloud_mask==1) {
```

Comment:
Does cloud_mask == 1 indicate cloud-free pixel? If yes, I would something like this:
```javascript
let is_cloud_free = sample.cloud_maks == 1 ? 1 : 0
return [factor * sample.red, factor * sample.green, factor * sample.blue, is_cloud_free]
```

Reply:
yes, added opacity to do that. thanks!